src/components/carpool/CarpoolRouteWaypointOptimizerWidget.tsx

```
@@ -0,0 +1,202 @@
+  { riderId: "r4", riderName: "David Miller", pickupLocation: "East Innovation Hub", lat: 37.7710, lng: -122.4050 },
+];
+
+export const CarpoolRouteWaypointOptimizerWidget: React.FC<CarpoolRouteWaypointOptimizerWidgetProps> = ({
+  carpoolId = "carpool-robotics-2026",
+  driverName = "Alex Rivera",
+  venueName = "Regional Tech Conference Center",
+  initialWaypoints = MOCK_WAYPOINTS,
+  onRouteOptimized,
+  className,
+}) => {
+  const [optimizationResult, setOptimizationResult] = useState<CarpoolRouteOptimizationResult>(() => {
```

ESLint

[error] Insert `⏎····`

(prettier/prettier)